skimage/segmentation/_clear_border.py

```diff
@@ -82,9 +82,9 @@ def clear_border(labels, buffer_size=0, bgval=0, in_place=False, mask=None,
 
     if mask is not None:
         err_msg = ("labels and mask should have the same shape but "
-                   "are {} and {}")
+                   f"are {out.shape} and {mask.shape}")
```

Comment:
We were discussing in https://github.com/scikit-image/scikit-image/pull/5478#pullrequestreview-708813111 whether all lines in a group should start with `f` in cases like this. I don't have a strong feeling either way, but we should do the same in this PR. Since we are still discussing, I would make the change yet.

Reply:
It also came up in that PR to potentially change to single quoted strings while we are already modifying these lines.